requirements/extra-pdf-image.in

```diff
@@ -3,7 +3,7 @@
 
 pdf2image
 pdfminer.six
-unstructured-inference
+unstructured-inference==0.5.28
```

Comment:
please add note that this is intentionally pinned here vs. contraints.in for the sake of the pip installable package. maybe an explicit hint to "do not move to constraints.in" :)